internal/store/x509_auth_database_users.go

```diff
@@ -24,21 +24,26 @@ import (
 
 //go:generate mockgen -destination=../mocks/mock_x509_certificate_store.go -package=mocks github.com/mongodb/mongocli/internal/store X509CertificateDescriber,X509CertificateSaver,X509CertificateStore
 
-type X509CertificateDescriber interface {
+type X509CertificateConfDescriber interface {
 	X509Configuration(string) (*atlas.CustomerX509, error)
 }
 
-type X509CertificateSaver interface {
+type X509CertificateConfSaver interface {
 	SaveX509Configuration(string, string) (*atlas.CustomerX509, error)
 }
 
 type X509CertificateDisabler interface {
 	DisableX509Configuration(string) error
 }
 
+type UserCertificateLister interface {
+	GetUserCertificates(string, string) ([]atlas.UserCertificate, error)
+}
+
 type X509CertificateStore interface {
-	X509CertificateDescriber
-	X509CertificateSaver
+	UserCertificateLister
```

Comment:
I would not mix these together (atlas managed/customer managed) but I don't feel strong about it

Reply:
Cool yeah i wasn't sure whether to keep them together because the client store ends up being the same, but I think separate will be easier